crates/smartmodule-development-kit/src/test.rs

```diff
@@ -165,6 +165,11 @@ impl TestCmd {
 
         let output = chain.process(sm_input, &metrics)?;
 
+        if let Some(error) = output.error {
+            print!("SmartModule error: {error}");
```

Comment:
The reasoning here is that `stderr` should be populated only if `process` returns a `Err`, leaving `stdout` as a way to tell that the SmartModule execution wasn't a success but the involving command acted as expected.

Regardless, it is possible to print to `stderr` using `eprint` if desired.

Reply:
Sounds good to me! I think we could have outputs like:

```
Failed to process 1 record with errors:

1. <Error Message>
```